ibis/backends/flink/registry.py

```diff
@@ -382,6 +389,11 @@ def _timestamp_from_ymdhms(
     return f"CAST({concat_string} AS TIMESTAMP)"
 
 
+def _struct_field(translator, op):
+    arg = translator.translate(op.arg)
+    return f"{arg}.`{op.field}`"
```

Comment:
Should this use `quote_identifier`?

Reply:
I took this code from BigQuery, but sure I can change it to `quote_identifier`.